lncrawl/templates/novelupdates.py

```diff
@@ -46,6 +46,18 @@ def cleanup_prompts(self):
         except Exception:
             pass
 
+    def relibrary_url_transformation(self, response, chapter: Chapter) -> str:
+        soup = BeautifulSoup(response.text, "lxml")
+        post_url = soup.select("div > p > a")[-1]["href"]
+        if "page_id" in post_url:
+            return post_url
+        novel_url = f"https://re-library.com/translations/{post_url.split('/')[4:5][0]}"
+        time.sleep(5)
```

Comment:
is this sleep required?

Reply:
I experimented with different urls and sleep values. This sleep is necessary if there are more than 50 transitions from novelupdates to re-library. Otherwise, the site rejects the connection.
Alternatively, we can set workers=2, but this will slow down the work for the rest of the sources.